### PR TITLE
style(footer): fix footer based on design spec

### LINKED ISSUE #73: [Footer] wrong color on footer and wrong shape on icon
## Description
1. footer 底色應為 ''#18152D''
2. copyright 底色應為 ''#0E0D1C''
3. twitter, youtube, mail logo 的形狀應為圓形
image

## Steps to Reproduce

1. Go to Index
2. Scroll down to footer
4. See an error

## Expected Behavior
1. footer 底色應為 ''#18152D''
2. copyright 底色應為 ''#0E0D1C''
3. twitter, youtube, mail logo 的形狀應為圓形
image

## Actual Behavior
1. footer 底色應為 ''#18152D''
2. copyright 底色應為 ''#0E0D1C''
3. twitter, youtube, mail logo 的形狀應為圓形
image
## More Information
可以在 Figma 上看到原始設計稿
figma連結：https://www.figma.com/file/1za4uYNpSonvgxdcLoHAzs/PyCon-Taiwan-2021-Web?node-id=166%3A0

Additional context

### PR DESCRIPTION
**Pull request by linooohon**
## Types of Changes

- [x] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description

Fixed #73

### CODE REVIEW

**Review by josix — CHANGES_REQUESTED** [review]

**Comment by josix** [comment]
Hello @linooohon, thanks for your hard work. There are two coding style nit and others LGTM.

**Comment by linooohon** [comment]
Hi @josix, thanks for your review. Here is the commit for this fix! [fded3f4](https://github.com/pycontw/pycontw-2021/pull/87/commits/fded3f49f43b47dfc6df9229b00d391a98ed0fe6)

**Review by josix — CHANGES_REQUESTED** [review]

File: components/core/footer/FooterIcon.vue [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+    <div class="core-footer">
+        <ext-link
+            v-for="(icon, index) in iconArray"
+            :key="`icon_${index}`"
+            :href="icon.link"
+            class="core-footerLink"
+        >
+            <img
+                :src="icon.src"
+                :alt="`${icon.src.split('/')[3].split('.')[0]} Footer Icon`"
```
[the remark this answers is not in the document]
Comment (linooohon) [reply components/core/footer/FooterIcon.vue]: Thanks for your advice, I agree with your point. [17f24fb](https://github.com/pycontw/pycontw-2021/pull/87/commits/17f24fb306945cfca234985482b17d37ba60c36b)

**Review by josix — APPROVED** [review]
LGTM, thank you @linooohon!

**Pull request merged**